cea/datamanagement/format_helper/cea4_verify.py

```
@@ -0,0 +1,402 @@
+        raise ValueError("Unsupported file type. Please provide a .csv or .shp file.")
+
+    # Find duplicate names
+    list_names_duplicated = df['name'][df['name'].duplicated()].tolist()
+
+    return list_names_duplicated
+
+
+def print_verification_results_4(scenario_name, dict_missing):
+
+    list_missing_files_shp_building_geometry = dict_missing.get('building-geometry')
+    list_missing_files_csv_building_properties = dict_missing.get('building-properties')
```

Ruff (0.8.2)

Local variable `list_missing_files_csv_building_properties` is assigned to but never used

Remove assignment to unused variable `list_missing_files_csv_building_properties`

(F841)